lntest/itest/lnd_channel_backup_test.go

```diff
@@ -847,6 +847,12 @@ func testChanRestoreScenario(t *harnessTest, net *lntest.NetworkHarness,
 	ctxt, _ := context.WithTimeout(ctxb, defaultTimeout)
 	net.SendCoins(ctxt, t.t, btcutil.SatoshiPerBitcoin, carol)
 
+	// For the anchor output case we need two UTXOs for Carol so she can
+	// sweep both the local and remote anchor.
+	if testCase.anchorCommit {
+		net.SendCoins(ctxt, t.t, btcutil.SatoshiPerBitcoin, carol)
```

Comment:
Do we need to wait for these to arrive? Or are they confirmed later in the test

Reply:
`SendCoins` actually mines 6 blocks to confirm them, so no extra wait is necessary.